src/main/java/com/sngular/annotation/processor/mapping/ShortMapping.java

```diff
@@ -29,18 +29,15 @@ public final String getFunctionType() {
 
   @Override
   public final String getFunctionOnlyValue() {
-    return "shortValue";
+    return "integerValue";
```

Comment:
Please, keep the right type

Reply:
If you check the documentation of pact you can see there is no shortType, that is why change to integerType that is the close one we got.

https://docs.pact.io/implementation_guides/jvm/consumer